apps/server/src/modules/account/controller/dto/patch-my-account.params.ts

```diff
@@ -43,6 +44,7 @@ export class PatchMyAccountParams {
 
 	@IsString()
 	@IsOptional()
+	@SanitizeHtml()
```

Comment:
you can add an api test also, to make sure nobody removes this in the future

Reply:
Added a test. Thanks for the hint!